addons/controllers/testdata/test-webhook-manifests.yaml

```diff
@@ -5,12 +5,8 @@ metadata:
   name: webhook-service
   namespace: tkg-system
 spec:
-  ports:
-    - port: 443
-      protocol: TCP
-      targetPort: 9443
-  selector:
-    control-plane: controller-manager
+  type: ExternalName
+  externalName: 127.0.0.1
```

Comment:
Trying to understand the `externalName: 127.0.0.1` here. Is it trying to redirect traffic to the webhook server started by mgr.GetWebhookServer()?
If so, I this we should probably explicitly configure mgr.GetWebhookServer() in suite_setup.

Reply:
Though it is the default setting, but we are still explicitly setting these values to avoid any side effects.